google_compute_engine_oslogin/utils/oslogin_utils_test.cc

```diff
@@ -267,8 +267,8 @@ TEST(ParseJsonEmailTest, FailsParseEmail) {
 
 TEST(ParseJsonSshKeyTest, ParseJsonToSshKeysSucceeds) {
   string test_user =
-      "{\"loginProfiles\":[{\"name\":\"jonesdl@google.com\",\"sshPublicKeys\":["
-      "{\"key\":\"test_key\"}]}]}";
+      "{\"loginProfiles\":[{\"name\":\"jonesdl@google.com\",\"sshPublicKeys\":"
```

Comment:
Please update this test to use a different email (e.g. foo@example.com).

I'll update more after we have completed the backend changes.